lib/install.js

```diff
@@ -81,7 +88,7 @@ async function install(targetPath, version) {
   }
 }
 
-module.exports = async function(version) {
+module.exports = async function(version,proxy) {
```

Comment:
you had a 3rd param `verifyssl` that you were passing to this?

Reply:
forget to include it in this branch, great catch thanks!

Reply:
added